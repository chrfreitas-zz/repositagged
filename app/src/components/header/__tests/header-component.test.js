import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Header from '../index';

describe('Header view', () => {
  it('should match with snapshot', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Header title="SejÃ¡ bem vindo" />);
    expect(result).toMatchSnapshot();
  });
});
