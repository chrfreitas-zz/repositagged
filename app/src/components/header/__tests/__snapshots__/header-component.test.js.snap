// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header view should match with snapshot 1`] = `
<div
  className="header"
>
  <div
    className="header__title"
  >
    Sej√° bem vindo
  </div>
  <div
    className="header__menu"
  >
    <Route
      component={[Function]}
      path="/:username/repositories"
    />
  </div>
</div>
`;
